bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -184,11 +184,14 @@ spec:
           configMap:
             name: {{ template "zookeeper.fullname" . }}
         {{- end }}
-        {{- if not .Values.persistence.enabled }}
+{{- if not .Values.persistence.enabled }}
         - name: data
           emptyDir: {}
-        {{- end }}
-  {{- if .Values.persistence.enabled }}
+{{- else if .Values.persistence.existingClaim }}
```

Comment:
I would revert the logic to be more understandable. For example:
```

       {{- if and .Values.persistence.enabled .Values.persistence.existingClaim }}
        - name: data
          persistentVolumeClaim:
            claimName: {{ .Values.persistence.existingClaim }}
       {{- else }}
       - name: data
          emptyDir: {}
       {{- end }}

Reply:
logic is reverted now - please mind that the original logic was copied from bitnami/kafka - so they differ now